release-scripts/getUpdatedVersions.js

```diff
@@ -0,0 +1,80 @@
+const { spawn } = require("child_process");
+const fs = require('fs');
+const process = require('process')
+
+async function runProcess(command, args) {
+    const p = spawn(command, args)
+    let output = '';
+    p.stdout.on("data", data => {
+        output += data.toString()
+    });
+
+
+    return new Promise((resolve, reject) => {
+        p.on('exit', (code) => {
+            if (code !== 0) {
+                return reject(code);
+            }
+
+            resolve(output)
+        })
+    });
+}
+
+const VERSION_DIFF_REGEX = /^\+\s*"version":\s*"([0-9\.]+)".*$/
+const VERSION_DIFF_OLD_REGEX = /^\-\s*"version":\s*"([0-9\.]+)".*$/
+
+function formatForGithubActions(markdownString) {
+    return markdownString.replace(/\n/g, "%0A");
+}
+
+function extractVersionFromLines(lines, regex) {
+    const [line] = lines
+            .filter(line => regex.test(line));
+            
+    const regexResult = regex.exec(line);
+
+    if (!regexResult || !regexResult[1]) {
+        throw new Error("Cannot parse version for module " + module) 
+    }
+
+    return regexResult[1];
+}
+
+async function getBumpedModules() {
+    const modules = fs.readdirSync('./lib');
```

Comment:
Would advise against any sync FS operations, and always use async operations. The `fs` module now supports promises: https://nodejs.org/api/fs.html#promises-api